utils/nctl/sh/scenarios/validators_disconnect.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/utils/main.sh
+source "$NCTL"/sh/views/utils.sh
+source "$NCTL"/sh/node/svc_"$NCTL_DAEMON_TYPE".sh
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+# Exit if any of the commands fail.
+set -e
+
+function main() {
+    log "------------------------------------------------------------"
+    log "Starting Scenario: Half of validators disconnecting"
+    log "------------------------------------------------------------"
+
+    # 0. Start the rest of the validator nodes
+    log_step "Starting nodes 6-10"
+    nctl-start node=6
+    nctl-start node=7
+    nctl-start node=8
+    nctl-start node=9
+    nctl-start node=10
+    # 1. Wait for the genesis era to complete
+    do_await_genesis_era_to_complete
+    # 2. Allow the chain to progress
+    do_await_era_change 1
```

Comment:
I think this step is not needed, we're already in era 2 after `do_await_genesis_era_to_complete`

Reply:
removed this step in https://github.com/casper-network/casper-node/pull/4120/commits/a980a885fe8d2f648071469ce4f53662b8b29326